upstream-community-operators/aws-service/aws-service.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,194 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: aws-service-operator.v0.0.1
+  namespace: placeholder
+  annotations:
+    categories: "Cloud Provider"
+    description: The AWS Service Operator allows you to manage AWS resources using Kubernetes Custom Resource Definitions.
+    containerImage: awsserviceoperator/aws-service-operator:v0.0.1-alpha3
+    createdAt: 2019-02-09T12:00:00Z
+    support: AWS Labs
+spec:
+  displayName: AWS Service Operator
+  description: >
+    The AWS Service Operator allows you to manage AWS resources using Kubernetes
+    Custom Resource Definitions. Using the AWS Service Operator enables a gitops
+    workflow to drive your infrastructure to the desired state leveraging
+    Kubernetes Custom Resource Definitions (CRD), the Kubernetes internal
+    control loop, and AWS CloudFormation orchestration.
+
+
+    In order to run the Operator the node will need AWS IAM privileges. This can
+    be done through a service like kube2iam. If you are using an Amazon Elastic
+    Container Service for Kubernetes (EKS) cluster, you most likely provisioned
+    this with CloudFormation.
+
+
+    The command shown below will use the CloudFormation stacks to try to update
+    the proper Instance Policy with the correct permissions. Make sure to
+    replace `${STACK_NAME}` with the nodegroup stack name from the
+    CloudFormation console. A more limited IAM role may be substituted if desired.
+
+
+        aws iam attach-role-policy --policy-arn arn:aws:iam::aws:policy/AdministratorAccess --role-name $(aws cloudformation describe-stacks --stack-name ${STACK_NAME} --output json | jq -r ".Stacks[0].Outputs[0].OutputValue" | sed -e 's/.*\///g')
```

Comment:
Should this line be surrounded by backticks, since it's a terminal command?
Eg:
```
aws iam ...
```

Reply:
Rob had posted the description into the UI and the code-blocks were present.